tests/puppeteer/.docker/prestashop/docker_run.sh

```diff
@@ -27,6 +27,9 @@ fi
 if [ $PS_DEV_MODE -ne 0 ]; then
   echo "\n* Enabling DEV mode ...";
   sed -ie "s/define('_PS_MODE_DEV_', false);/define('_PS_MODE_DEV_',\ true);/g" /var/www/html/config/defines.inc.php
+else
+  echo "\n* Disabling DEV mode ...";
+  sed -ie "s/define('_PS_MODE_DEV_', true);/define('_PS_MODE_DEV_',\ false);/g" /var/www/html/config/defines.inc.php
```

Comment:
Since this docker_run.sh is only for puppeteer, you can directly do 
`sed -ie "s/define('_PS_MODE_DEV_', true);/define('_PS_MODE_DEV_',\ false);/g" /var/www/html/config/defines.inc.php` and remove the condition about PS_DEV_MODE :)

Reply:
Yes no need to :+1: